lib/utils.ts

```diff
@@ -77,15 +50,15 @@ export const mergeObjects = _.merge;
  */
 export const evaluateSetting = <T>(
 	settings: object | undefined | null = {},
-	property: string,
+	property: string
 ): T => {
-	let value = _.get(settings, property);
+	let value = get(settings, property);
 
 	if (value == null) {
 		throw new Error(`Setting not found: ${property}`);
 	}
 
-	if (_.isFunction(value)) {
+	if (typeof value === "function") {
```

Comment:
😱😱😱!!! double quotes !!!😱😱😱
```suggestion
	if (typeof value === 'function') {
```
I guess we don't use balena-lint here right?

Reply:
I just saved and it formatted to have the single quotes so I guess it was just previously committed without formatting 🤔